src/navigators/createNavigator.js

```diff
@@ -36,12 +37,23 @@ function createNavigator(NavigatorView, router, navigationConfig) {
       return this.props.navigation;
     };
 
+    _dangerouslyUpdateRouter = (key, newRouteConfigs) => {
+      if (router.setRoutes) {
```

Comment:
After the action creators overhaul, I don't think there would be a way to access the router from here.

Reply:
this line is actually not needed because it's just using dispatch anyhow